core/evancore.vue.libs/src/components/table/table.scss

```diff
@@ -183,6 +180,7 @@ $table-row-height: 56px;
 
       td, th {
         padding: 8px;
+        vertical-align: initial !important;
```

Comment:
Please don't do this. Initial is handled by the browser and could be theoretically everything the vendor puts in.
What do you want to achieve with this rule?

Reply:
I simply want to reset a bootstrap / evancore style. Initial is the same as "baseline" for this property (https://developer.mozilla.org/de/docs/Web/CSS/vertical-align)

Reply:
it's not guaranteed, that every browser applies baseline here. Why not use baseline instead of initial?